nova/db/sqlalchemy/migrate_repo/versions/392_fix_internal_access_path_max_length.py

```diff
@@ -0,0 +1,30 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from sqlalchemy import MetaData, Text, Table
+
+
+def upgrade(migrate_engine):
+    meta = MetaData()
+    meta.bind = migrate_engine
+
+    table_console_auth_tokens = Table('console_auth_tokens',
+                                      meta,
+                                      autoload=True)
```

Comment:
pep8 tests are broken. I can't open them, but this looks like a possible underindention.

Reply:
Hmm, don't see any pep8 failures, but there is a test which says that the tests for this migration are missing. Will add one.

Reply:
Finally "all green".